storage/badger_storage.go

```diff
@@ -565,19 +561,19 @@ func BadgerTrain(
 			entriesSeen++
 
 			if entriesSeen > maxEntries-1 && maxEntries != -1 {
-				return errMaxEntries
+				return ErrMaxEntries
 			}
 
 			return nil
 		},
 		true,
 	)
-	if err != nil && !errors.Is(err, errMaxEntries) {
+	if err != nil && !errors.Is(err, ErrMaxEntries) {
 		return -1, -1, fmt.Errorf("%w: unable to scan for %s", err, namespace)
```

Comment:
nit: do we need to extract this error message into it's own const as well?

Reply:
good catch, I had taken it out while I fixed breaking tests but can add it back in now!